size_matters/experiments.py

```diff
@@ -37,16 +38,14 @@ def compare_methods(dataset: Dataset, max_voters: int, n_batch: int) -> None:
 
     # initialize the accuracy array
     accuracy = np.zeros([5, n_batch, max_voters - 1])
-    for num in range(1, max_voters):
-        print("Number of Voters :", num)
-        for batch in range(n_batch):
-            print("###### Batch ", batch, " ######")
-
+    for num in tqdm(
+        range(1, max_voters), desc="Number of voters", position=0, leave=True
+    ):
+        for batch in tqdm(range(n_batch), desc="Batch", position=1, leave=False):
             # Randomly sample num voters
+
             voters = sample(list(annotations[COLUMNS.voter].unique()), num)
-            annotations_batch = annotations[
-                annotations[COLUMNS.voter].isin(voters)
-            ]
+            annotations_batch = annotations[annotations[COLUMNS.voter].isin(voters)]
```

Comment:
These are formatting changes (should have been seperated into their own commit but let's don't mind them this time)

Reply:
btw we should agree on a common formatting rules. In fact this shows that we're not using the same --line-length for the black formatter. The default is 88. I'll find a way to make it common to all contributors.